compiler/src/iree/compiler/Codegen/Common/PolynomialApproximationPass.cpp

```diff
@@ -30,6 +30,7 @@ class PolynomialApproximationPass
     RewritePatternSet mathPatterns(&getContext());
     populateExpandTanPattern(mathPatterns);
     populateExpandExp2FPattern(mathPatterns);
+    populateExpandPowFPattern(mathPatterns);
```

Comment:
It's not clear to me what qualifies the expansions to be here or in the if-else statement below... but this should be similar to Exp2

Reply:
IIUC, it should be inside if-then-else if you are worried the approximation is not precise enough. As far as I know, if exp2 is precise enough then this shouldn't cause a huge issue.